source/material_model/rheology/drucker_prager.cc

```diff
@@ -29,6 +29,29 @@ namespace aspect
   {
     namespace Rheology
     {
+      template <int dim>
+      double
+      DruckerPrager<dim>::compute_steady_state_friction (const double angle_internal_friction,
+                                                         const double angle_internal_friction_dynamic,
+                                                         const double effective_strain_rate,
+                                                         const double reference_strain_rate) const
+      {
+
+        // Calculate effective steady-state friction coefficient (mu). The formula below is equivalent to the
+        // equation 13 in van Dinther et al., (2013, JGR). Although here the dynamic friction coefficient
```

Comment:
Please copy the sentence about the publication into the header as well. That is something that is interesting even for readers who are not interested in how it is implemented exactly (=for ASPECT), but only want to understand what this function does in general.